3-JSS/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const CopyPlugin = require('copy-webpack-plugin')
+module.exports = {
+    mode: 'development',
+    entry: {
+        main: path.resolve(__dirname, 'src/webpackTodoList.js')
+    },
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: '[name].[contenthash].js',
+        clean: true,
+    },
+    devtool: 'eval-source-map',
```

Comment:
what does this do, what are the other options?

Reply:
regarding source maps, there are several of them and each one of them refer to prod or development and the time it takes etc.